docs.md

```diff
@@ -0,0 +1,40 @@
+## General Moments
```

Comment:
Please follow the file structure outlined in issue description: docs directory, Architecture.md file within this directory
We don't want to put too many files in repository root, and docs.md seems too generic